weedcoco/index/indexing.py

```diff
@@ -3,43 +3,52 @@
 import pathlib
 import json
 import requests
-from math import ceil
+import urllib
 
 
 class ElasticSearchIndex:
+
+    """
+    Parameters:
```

Comment:
What docstring standard is this following? It's almost [Google Style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), but there they would have:
```suggestion
    Args:
```

And in weedcoco/importers/voc.py we have [one function](https://github.com/Sydney-Informatics-Hub/Weed-ID-Interchange/blob/d7ae6ba709e9652999000674f9ca3655c9ebb10a/weedcoco/importers/voc.py#L41-L49) with numpydoc-style docstrings. I'm happy to change it if you prefer Google style, but we should be consistent.